src/styles/block/calendar__block.scss

```
@@ -0,0 +1,25 @@
+.calendar__block {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  width: $square-block;
+  height: $square-block;
+  border: 1px solid black;
+  background-color: #eee;
+  transition-duration: 0.5s;
```

indicate which properies you use transition for
```suggestion
  transition: background-color 0.5s, transform 0.5s;
```